### DISCUSSION TITLE
ch1-basic/ch1-05-mem

### DISCUSSION THREAD

**Discussion by giscus** [chai2010/advanced-go-programming-book#609] [General]
# ch1-basic/ch1-05-mem

Go语言高级编程

https://chai2010.cn/advanced-go-programming-book/ch1-basic/ch1-05-mem.html

**Comment by Engrtao** [comment]
1.5.3 咋感觉反直觉呢，实际实验，能正确输出啊

**Reply by chenwangnatsukashii** [reply]
感觉Go中是可见的，但是Java没有volatile就不行吧

**Reply by J-Corleone** [reply]
有概率的，但对于这样一个简单的程序来说很小
